FORMAT_ADIOS.md

```diff
@@ -0,0 +1,30 @@
+Implementation Details: ADIOS1/2
+================================
+
+Datatype Conventions
+--------------------
+
+## Boolean Types
+
+ADIOS1 (BP3) was traditionally a C library and as such as no notion for boolean types.
+[At the moment](https://github.com/ornladios/ADIOS2/issues/2606), also ADIOS2 (BP4) does not define boolean types.
+Although the openPMD-standard avoids using boolean types itself, developers might choose to implement them as attributes or dataset types.
+
+For compatibility, boolean types shall be stored as `adios_unsigned_byte`/`unsigned char`.
+
+### ADIOS1
+
+ADIOS1 is deprecated.
+A stored boolean type is identified as `adios_unsigned_byte` on the read side, since no convention for bool-identification is specified.
+
+### ADIOS2
+
+For each boolean attribute or variable, an additionally attribute is stored adding identification for bool for data readers.
+These identifying attributes are prefixed with `__is_boolean__/` (for attributes) or `__is_boolean_var__/` (for variables), and then adding the full path of the described attribute or variable.
```

Comment:
@franzpoeschel I am already defining the identifier for ADIOS2 boolean variables here as well: https://github.com/openPMD/openPMD-api/issues/921

Reply:
I think it was a bit unlucky on my side that I used `__openPMD_internal/useSteps` but did not put the boolean attributes in `__openPMD__internal/` as well. We could maybe try coupling this with the new attribute layout and put the boolean identifiers there once we switch to it?

Reply:
Discussed on slack: in this PR, we document the current (openPMD 1) layout and will open a new pull request once the new ADIOS2 layout is finalized (https://github.com/openPMD/openPMD-api/issues/920).

At the moment we have not yet established a convention for boolean round-trip representation for data sets (ADIOS variables) and we will either establish one: https://github.com/openPMD/openPMD-api/issues/921 or hopefully can rely on a future ADIOS2 bool type: https://github.com/openPMD/openPMD-api/issues/921